src/Orleans/GrainDirectory/MultiClusterStatus.cs

```diff
@@ -6,7 +6,22 @@
     internal enum MultiClusterStatus : byte
```

Comment:
I keep tripping over this enum name while reviewing the code. It reads as if it represents state of the multi-cluster. Maybe GrainDirectoryEntryStatus instead?